background.js

```diff
@@ -2,29 +2,16 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-async function sendPing() {
-  const currentDate = new Date();
-
-  const payload = {
-    datetime: currentDate,
-  };
-
-  const options = {
-    schemaName: "debug",
-    schemaVersion: 1,
-  };
-
-  // FIXME pass messages to core add-on
-  throw new Error("not implemented");
-}
-
 let ion = new Ion();
-ion.initialize();
-
-sendPing()
-  .then((result) =>
-    console.info(
-      "Telemetry submitted, check about:telemetry archived ping data."
-    )
-  )
-  .catch((error) => console.error("Could not send ping:", error));
+ion.initialize(
+  // A sample key id used for encrypting data.
+  "sample-invalid-key-id",
```

Comment:
Have you thought about how to load this into the extension? I wonder if we should read from a JSON file or something so the key doesn't need to be hardcoded.

Reply:
Also why is the default `sample-invalid-key-id` is this for testing elsewhere?

Reply:
> 
> 
> Have you thought about how to load this into the extension? I wonder if we should read from a JSON file or something so the key doesn't need to be hardcoded.

Unfortunately, Chromium based browsers throw when there's unknown keys in the manifest, so it can't live there :( 
I think we can evaluate other options, as needed! I just wanted to lay the foundations.

> 
> 
> Also why is the default `sample-invalid-key-id` is this for testing elsewhere?

If we don't provide sample values, then `ion.initialize` would fail and not initialize. I thought providing sample (even if useless) values with some clear comments was a better option, but I have no strong opinion.